packages/makesheet/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "makesheet",
+  "version": "0.1.0",
+  "description": "Google Sheets timestamp service for Make-based workflows",
+  "main": "sheetInfo.js",
+  "author": "Dan Connolly",
+  "license": "MIT",
+  "scripts": {
+    "push": "clasp push",
+    "deploy": "clasp push; clasp deploy --deploymentId $DEPLOYMENT_ID",
+    "curl": "curl -L https://script.google.com/macros/s/$DEPLOYMENT_ID/exec?sheetId=$SHEET_ID",
+    "login": "clasp login"
+  },
+  "devDependencies": {
+    "@google/clasp": "^2.3.0",
+    "@types/google-apps-script": "^1.0.17",
+    "typescript": "^4.0.5"
+  }
+}
```

Comment:
missing newline

Reply:
or maybe it does:

```console
finquick/packages/makesheet$ tail -c1 package.json | od -An -t x1
 0a
 ```
 
 odd. both github and emacs show it without one.

Maybe I got confused about which file somehow.